redis-utils-lib/tests.edn

```diff
@@ -0,0 +1,24 @@
+#kaocha/v1
+{:tests [{:id :unit
+          :ns-patterns ["cmr.redis-utils.test*" "-test$"]}]
+
+ :plugins [:kaocha.plugin.alpha/info
+           :kaocha.plugin/cloverage
+           :capture-output
+           :junit-xml
+           :notifier
+           :print-invocations
+           :profiling]
+
+ :fail-fast? false
+
+ :color? #profile {:default true
+                   :ci false}
+
+ :reporter #profile {:default kaocha.report.progress/report
+                     :ci kaocha.report/documentation}
+
+ :kaocha.plugin.junit-xml/target-file "target/junit.xml"
+
+ :kaocha.plugin.profiling/count 3
```

Comment:
I've seen this in all of the `tests.edn` files and it's kind of a magic number. Maybe a comment about why 3 is the right number here would be useful